File: udgaard/src/main/java/com/skrymer/udgaard/model/MarketSymbol.java
package com.skrymer.udgaard.model;

public enum MarketSymbol {
    FULLSTOCK("FULLSTOCK"),
    XLE("Energy"),
    XLV("Health"),
    XLB("Materials"),
    XLC("Communications"),
    XLK("Technology"),
    XLRE("Realestate"),
    XLI("Industrials"),
    XLF("Financials"),
    XLY("Discretionary"),
    XLP("Staples"),
    XLU("Utilities"),
    UNK("Unknown");

    private String description;

    private MarketSymbol(String description){
        this.description = description;
    }

    public String getDescription() {
        return description;
    }
}

File: new_udgaard/udgaard/src/main/kotlin/com/skrymer/udgaard/model/MarketSymbol.kt
package com.skrymer.udgaard.model

enum class MarketSymbol(description: String) {
    FULLSTOCK("FULLSTOCK"),
    XLE("Energy"),
    XLV("Health"),
    XLB("Materials"),
    XLC("Communications"),
    XLK("Technology"),
    XLRE("Realestate"),
    XLI("Industrials"),
    XLF("Financials"),
    XLY("Discretionary"),
    XLP("Staples"),
    XLU("Utilities"),
    UNK("Unknown");

    val description: String?

    init {
        this.description = description
    }
}
